### PR TITLE
Adding x86 UWP F5 test runs to Helix

### PR DESCRIPTION
**Pull request by joperezr**
Adding x86 test runs for UAP F5 to Helix.

cc: @danmosemsft @MattGal @safern @tarekgh @AlexGhiondea @weshaggard @joshfree

### CODE REVIEW

File: external/test-runtime/XUnit.Runtime.depproj [hunk 1]
```diff
@@ -126,7 +126,7 @@
     <PropertyGroup>
       <UAPToolsPackageName>microsoft.dotnet.uap.testtools</UAPToolsPackageName>
 
-      <UAPToolsFolder Condition="'$(UAPToolsFolder)'==''">$(PackagesDir)$(UAPToolsPackageName)\$(UAPToolsPackageVersion)\Tools</UAPToolsFolder>
+      <UAPToolsFolder Condition="'$(UAPToolsFolder)'==''">$(PackagesDir)$(UAPToolsPackageName)\$(UAPToolsPackageVersion)\Tools\$(ArchGroup)</UAPToolsFolder>
```
Comment (MattGal) [inline external/test-runtime/XUnit.Runtime.depproj]: Is there a complementary change that also changes the folder structure here or was this already like this?

**Comment by joperezr** [comment]
I ran a Debug-x86 private run and things look good, around 200 test failures which is around the expected ones.

**Comment by joperezr** [comment]
@MattGal https://mc.dot.net/#/user/joperezr/pr~2Fcorefx~2Fmaster~2F/test~2Ffunctional~2Fuwp~2F/20170619.2449

File: external/test-runtime/XUnit.Runtime.depproj (hunk 1 above)
Comment (joperezr) [inline external/test-runtime/XUnit.Runtime.depproj]: complementary change on a package that is already in myget

**Pull request merged**